src/Menu/MenuItemGroup.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import {GroupLabel} from '../MenuList';
+
+const Wrapper = styled.span`
+	width: 100%;
+`;
+
+const MenuItemGroup = ({label, id = label, children}) => {
+	return (
+		<Wrapper aria-labelledby={id} role="group">
+			<GroupLabel id={id} role="separator">
```

Comment:
The label shouldn't have a role of separator, I think that would render the `{label}` inaccessible.

```suggestion
			<GroupLabel id={id}>
```